src/webserver/project/index.php

```diff
@@ -9,52 +9,55 @@
-			align-content: center;
-			align-items: center;
-            white-space: pre-line;
-            display: inline-block;
-		}
-		form {
-			margin: 0 auto;
-			width: 500px;
-    	}
-	</style> 
+	<link rel="stylesheet" href="project.css">
+	<link rel="stylesheet" href="../buttons.css" type="text/css">
+	<link rel="stylesheet" href="../font-size.css" type="text/css">
+
     <script type="text/javascript"></script>
 </head>
 
 <body>
 	<main>
-		<h1 style="text-align: justify;">Projekt Verwaltung</h1>
-		<form name="RegForm" id="registrierung" method="post">                                
-        	<div>	
-				<h2>Projekte:</h2>
-				<ul>
-					<?php					
-						$result = $connection->query("SELECT * FROM Project Order By ProjectName;");
-						while ($row = $result->fetch_object()) 
-						{						
-							echo "<div class='projectname' data-project-id='" . $row->ProjectId . "'>" . $row->ProjectName . "</div><br/>";				
-						}					
-					?>
-				</ul>
-				<h2>Mitarbeiter:</h2>	
-				<ul>
-					<?php					
-						$result = $connection->query("SELECT * FROM User Order By Username;");
-						while ($row = $result->fetch_object()) 
-						{						
-							echo "<input type='checkbox' class='user-checkbox' data-user-id='" . $row->UserId . "'>" . $row->Username . "</input><br/>";				
-						}					
-					?>
-				</ul>	
-				<a href="newproject"><button id="btn-new-projects" type="button">Projekt erstellen</button></a>
-				<a href=".."><button id="back-button" type="button">Zurück</button></a>			
-			</div>	
-		</form>    
+		<h1>Projekt Verwaltung</h1>
+		<form name="RegForm" id="registrierung" method="post">  
+			<div id="content-container">                              
+				<div id="projekte-container">	
+					<h2>Projekte:</h2>
+					<ul>
+						<?php					
+							$result = $connection->query("SELECT * FROM Project Order By ProjectName;");
+						
+							while ($row = $result->fetch_object()) 
+							{						
+								echo "<div class='projectname' data-project-id='" . $row->ProjectId . "'>" . $row->ProjectName . "</div><br/>";				
+							}					
+						?>
+					</ul>
+				</div>
+				<div id="mitarbeiter-container">
+					<h2>Mitarbeiter:</h2>	
+					<ul>
+						<?php					
+							$result = $connection->query("SELECT * FROM User Order By Username;");
+							while ($row = $result->fetch_object()) 
+							{						
+								echo "<input type='checkbox' class='user-checkbox' data-user-id='" . $row->UserId . "'>" . $row->Username . "</input><br/>";				
+							}					
+						?>
+					</ul>			
+				</div>	
+			</div>
+		</form>  
+		<div id="button-container">
+			<div id="button-width-container"> 
+				<!-- neuen wrapper auf 100% und dann kind auf flex und500px und vllt wieder flex-->
+				<div id="project-button-container">
+					<a href="newproject"><button id="btn-new-projects" type="button">Projekt erstellen</button></a>
+					<a href="deleteproject"><button id="btn-delet-project" type="button">Projekt löschen</button></a>
```

Comment:
```suggestion
					<button id="btn-delete-project" type="button">Projekt löschen</button>
```
Typo und ich glaube einen Link auf eine neue Seite brauchen wir dafür nicht. Das wird später mit JavaScript gemacht